api/client/client.go

```diff
@@ -86,15 +87,23 @@ func (c *Client) Close() error {
 }
 
 // NewDB creates a new DB with ID
-func (c *Client) NewDB(ctx context.Context, id thread.ID) error {
-	_, err := c.c.NewDB(ctx, &pb.NewDBRequest{
-		DbID: id.String(),
+func (c *Client) NewDB(ctx context.Context, creds thread.Credentials) error {
```

Comment:
Both these new DB methods now take credentials instead of a plain ID. Most APIs that took ID now take credentials. See below for what credentials are.